.changelog/982.txt

```diff
@@ -1,3 +1,3 @@
 ```release-note:bug
-Crash: Add StateMigrator for pass_credentials and wait_for_jobs (Inconsistent Plan)
+`resource/helm_release`: Fix an issue when the provider crashes with the error message `Provider produced inconsistent final plan`.
```

Comment:
AFAIK there are a few different issues manifested as "Provider produced inconsistent final plan". This entry might inadvertently lead some users to believe some of those other issues might be fixed as well.

Reply:
Thanks! Updatated.